development/cpp/configuring_an_ide/visual_studio_code.rst

```diff
@@ -61,7 +63,8 @@ Importing the project
         // Use this when your default shell is Command Prompt (cmd.exe).
         "-j %NUMBER_OF_PROCESSORS%",
         // Use this when your default shell is PowerShell.
-        "-j $env:NUMBER_OF_PROCESSORS"
+        "-j $env:NUMBER_OF_PROCESSORS",
+        "dev_build=yes",
```

Comment:
the image below also needs to be adjusted. (https://docs.godotengine.org/en/latest/_images/vscode_3_tasks.json.png)
If you can't, I can do that in a follow up since I'm using VSCode anyway. :)

Reply:
Updated, just needed to figure out how to toggle dark-mode :grin: